src/main/java/net/imglib2/RealRandomAccess.java

```diff
@@ -39,7 +39,7 @@
  * @author Stephan Preibisch
  * @author Stephan Saalfeld
  */
-public interface RealRandomAccess< T > extends RealLocalizable, RealPositionable, Sampler< T >
+public interface RealRandomAccess< T > extends RealLocalizable, RealPositionable, RandomAccess<T>
```

Comment:
Note that this line makes a `RealRandomAccess< T >` a `Localizable`, which it was not before. It's interesting that:
* `Localizable` extends `RealLocalizable` (i.e. if you can report your position in discrete space, then you can also report your position in real space)
* `RealPositionable` extends `Positionable` (i.e. if you can be positioned in real space, then you can also be positioned in discrete space)

Is there any historical reason for this?

Reply:
Unfortunately yeah, because the `(Real)RandomAccess` interfaces go both ways (read and write), so neither can extend the other with violating one or the other of the two above directions.

I think that's OK, though:
* `RealRandomAccess` implements `RealLocalizable` and `RealPositionable` which means it also implements `Positionable` – i.e. you can position a `RealRandomAccess` at an integer coordinate; and
* `RandomAccess` implements `Localizable` and `Positionable` which means it also implements `RealLocalizable` – i.e. you can query a `RandomAccess` in real space if you like.

So maybe no change needed for the RA/RRA interfaces?

Reply:
> So maybe no change needed for the RA/RRA interfaces?

By RA/RRA, you mean `RandomAccessible`/`RealRandomAccessible` (i.e. not the `Access`es?)